protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1276,7 +1276,7 @@ message ExportLedgerEntriesRequest {
   // Restrict exported ledger entries to those relating to the given party ID.
   string party_id = 1 [(google.api.field_behavior) = REQUIRED];
   // Restrict exported ledger entries to those relating to the given asset ID.
-  string asset_id = 2 [(google.api.field_behavior) = REQUIRED];
+  optional string asset_id = 2;
```

Comment:
Is this already reflected in the GraphQL?

Reply:
good question, but you can't access this API via GraphQL, it's for bulk downloading CSV files of ledger entries

Reply:
Ah, forgot that! Thank you :)